### PR TITLE
Adds initial files for BLE start guide

### PR DESCRIPTION
**Pull request by johnnyman727**
- [x] Add BLE usage content
- [x] Update images/gifs

### CODE REVIEW

**Comment by Frijol** [comment]
@johnnyman727 can you screenshot & gif an output from the example?

File: modules/ble.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+<div class="large-12 columns">
+
+### Step 3
+
+Make sure your Tessel is connected to a wifi network as shown on the [wifi page](/wifi.html). Then gain root access to the Tessel shell:
+
+`t2 root`
+
+This command gives you direct access to the Linux system running on Tessel. We need this in order to power up the USB dongle. Run this command:
+`hciconfig hci0 up`
```
Comment (HipsterBrown) [inline modules/ble.md:44]: Could this be a `t2` command eventually? Something like:

`t2 usb --on`

And does `hciconfig hci0 up` work for powering up all USB devices?

Comment (kevinmehall) [inline modules/ble.md:44]: `hciconfig` is only for Bluetooth. It should be made to happen automatically when a Bluetooth device is attached.

"power up" is not necessarily the right term. T2 does have the ability to fully cut power to USB devices under software control, but that's separate from enabling a Bluetooth HCI interface.

Comment (HipsterBrown) [inline modules/ble.md:44]: So we could change the command to `t2 ble` if enabling the interface automatically isn't possible.

Comment (johnnyman727) [inline modules/ble.md:44]: Fortunately, https://github.com/tessel/openwrt-tessel/pull/37 does this automatically (except for when the dongle is plugged in at boot, I still have to fix that).

**Pull request merged**